steps/applivery-deploy-ios/0.1/step.yml

```diff
@@ -0,0 +1,119 @@
+title: Applivery.com iOS Deploy
+summary: Deploy your awesome iOS Application to Applivery.com
+description: |-
+  Deploy an iOS application to [Applivery](http://www.applivery.com),
+    add notes and notify testers.
+
+    Register a Applivery account at [http://www.applivery..com/](http://www.applivery.com)
+    and create an App to utilize this step.
+
+    You also need to get your *Account API Key* for you account and the *App Id* for the app.
+website: https://github.com/applivery/steps-applivery-ios-deploy
+source_code_url: https://github.com/applivery/steps-applivery-ios-deploy
+support_url: https://github.com/applivery/steps-applivery-ios-deploy/issues
+published_at: 2016-02-07T16:44:09.038396058+01:00
+source:
+  git: https://github.com/applivery/steps-applivery-ios-deploy.git
+  commit: aca3a18df31de473bfc1c24b49a3d9d05a275c8e
+host_os_tags:
+- osx-10.10
+project_type_tags:
+- ios
+type_tags:
+- deploy
+- Applivery
+dependencies:
+- manager: brew
+  name: git
+is_requires_admin_user: false
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- ipa_path: $BITRISE_IPA_PATH
+  opts:
+    description: ""
+    is_dont_change_value: false
+    is_expand: true
+    is_required: true
+    summary: ""
+    title: IPA file path
+- api_token: ""
+  opts:
+    description: |-
+      This is the API Key to access your account.
+
+      ## Where to get the Applivery Account API Key?
+
+      Sign in to your [Applivery.com](http://dashboard.applivery.com) account,
+      click on Developers menu option from the left side menu and copy it from the
+      Account API Key section.
+    is_dont_change_value: false
+    is_expand: true
+    is_required: false
+    summary: ""
+    title: Account API Key
+- app_id: ""
+  opts:
+    description: |-
+      This is the App Id that identifies your App in Applivery.com
+
+      ## Where to get the App Id?
+
+      Sign in to your [Applivery.com](http://dashboard.applivery.com) account,
+      click on Applications menu option from the left side menu, click on the desired App.
+      You'll find the App Id inside the (i) information block (written in red).
+    is_dont_change_value: false
+    is_expand: true
+    is_required: false
+    summary: ""
+    title: 'Applivery: App ID'
+- notes: Deployed with Bitrise Applivery.com iOS Deploy Step.
+  opts:
+    description: Additional build/release notes
+    is_dont_change_value: false
+    is_expand: true
+    is_required: false
+    summary: ""
+    title: Notes attached to the deploy
+- notify: true
```

Comment:
It's better to specify boolean values as string ("true", "false"), because some yml parser cast this bools to yes/no, which will be used as string ("yes"/"no").